README.md

```diff
@@ -5,6 +5,17 @@ An Emacs major mode for [ReasonML](https://reasonml.github.io/).
 
 ## Installation
 
+### Prerequisites
+
+**Note**: the following setup assumes Reason and Merlin are installed. This can be achieved by by installing them from OPAM (`opam install reason merlin`). Make sure you're on ocaml 4.06.1!
```

Comment:
Technically 4.06 is not required. The reason syntax works with other versions of ocaml too. 4.06 is needed only for bucklescript

Reply:
Ok I will correct that. 

[Bucklescript documentation(https://bucklescript.github.io/docs/en/concepts-overview) still states that it is made for 4.02.3, I'll see if I can find a better source.